modules/apps/frontend-js/frontend-js-web/test/liferay/delegate/delegate.es.js

```diff
@@ -161,4 +161,24 @@ describe('delegate', () => {
 
 		expect(listener).not.toHaveBeenCalled();
 	});
+
+	it('gives access to delegateTarget inside the callback', () => {
```

Comment:
Nice!

Reply:
Yea, that was mostly Greg, but he wouldn't have gotten there if I didn't ask him about it!